packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -43,25 +42,65 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, ButtonVa
 
       // rectangular button defaults
       ...(!p.text && {
-        outline: 0,
-        borderWidth,
         borderStyle: 'solid',
         borderColor: v.borderColor,
         boxShadow: v.boxShadow,
+        borderRadius: '.2rem',
+        border: 0,
+
+        '::after': {
```

Comment:
what is the `after` and `before` here for?

Reply:
This is for applying the correct focus border styles to the buttons, one inner border and one outer border. When we use the normal border property, the content can shift, this just applied pseudo borders so the shape doesn't change.